src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -57,20 +60,31 @@ def __init__(
         :param settings: The inmanta environment settings that should be set on the remote orchestrator
         :param noclean: Option to indicate that after the run clean should not run. This exposes the attribute to other
                         fixtures.
+        :param ssl: Option to indicate wether SSL should be used or not. Defaults to false
+        :param token: Token used for authentication
+        :param ca_cert: Certificate used for authentication
         """
         self._env = environment
         self._host = host
         self._ssh_user = ssh_user
         self._ssh_port = ssh_port
         self._settings = settings
         self.noclean = noclean
+        self._ssl = ssl
+        self._token = token
+        self._ca_cert = ca_cert
 
         inmanta_config.Config.load_config()
         inmanta_config.Config.set("config", "environment", str(self._env))
-        inmanta_config.Config.set("compiler_rest_transport", "host", host)
-        inmanta_config.Config.set("compiler_rest_transport", "port", "8888")
-        inmanta_config.Config.set("client_rest_transport", "host", host)
-        inmanta_config.Config.set("client_rest_transport", "port", "8888")
+
+        for section in ["compiler_rest_transport", "client_rest_transport"]:
+            inmanta_config.Config.set(section, "host", host)
+            inmanta_config.Config.set(section, "port", "8888")
+
+            # Config for SSL and authentication:
+            inmanta_config.Config.set(section, "inm_ssl", ssl)
+            inmanta_config.Config.set(section, "inm_token", token)
+            inmanta_config.Config.set(section, "inm_ca_cert", ca_cert)
```

Comment:
The above-mentioned code sets incorrect config options. You can find the correct values: 
* https://github.com/inmanta/inmanta-core/blob/4312f9d0c124c1cb710c9039c05afce52f5af1b6/tests/conftest.py#L712
Or
* https://docs.inmanta.com/community/dev/reference/config.html#client_rest_transport.ssl